frontend/components/markdown/MarkdownEditor.vue

```diff
@@ -138,6 +138,10 @@ export default class MarkdownEditor extends Vue {
     }
 }
 
+.edit-btn i {
+    color: white !important;
+}
+
```

Comment:
I don't think this has to exist at all. if you want to change the color of the icon inside, use the color prop on the v-icon component, or better yet, the `light` and `dark` props that exist on almost all vuetify components

Reply:
the `light` and `dark` props did not work. However, together with some other CSS changes, I made it work with `color="white"`